kuma/wiki/tests/conftest.py

```diff
@@ -7,6 +7,11 @@
 from ..models import Document, Revision
 
 
+class Object(object):
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
I like ``namedtuple`` a lot as well, and it's lighter weight anyway (in terms of memory usage, not that it matters here), so I'll change to that.